frontend/packages/ceph-storage-plugin/src/components/ocs-install/pvs-available-capacity.tsx

```diff
@@ -15,18 +15,14 @@ export const PVsAvailableCapacity: React.FC<PVAvaialbleCapacityProps> = ({
 }) => {
   const { t } = useTranslation();
 
-  const [data, loaded, loadError] = useK8sWatchResource<K8sResourceKind[]>(pvResource);
+  const [data, loaded] = useK8sWatchResource<K8sResourceKind[]>(pvResource);
   let availableCapacity: string = '';
 
   let availableStatusElement = (
     <div className="skeleton-text ceph-pvs-available-capacity__current-capacity--loading" />
   );
 
-  if ((loadError || data.length === 0 || !storageClass) && loaded) {
-    availableStatusElement = (
-      <div className="text-muted">{t('ceph-storage-plugin~Not Available')}</div>
-    );
-  } else if (loaded) {
+  if (loaded && !!getName(storageClass)) {
```

Comment:
We should also check for `loadError` since when `loadError` is true, loaded` is also `true`.
In that scenario, when you have a pre selected storage class, it will pass this check.